docs/about/modules/ROOT/pages/faq.adoc

```diff
@@ -130,13 +129,15 @@ standby transactors. Centralised execution of transaction functions is
 effectively an optimisation that is useful for managing contention whilst
 minimising external complexity, and the trade-off is that the use of
 transaction functions will ultimately impact the serialised transaction
-throughput of the entire system. Crux does not currently provide a standard
-means of creating transaction functions but it is an area we are keen to see
-explored. If transaction functions and other kinds of validations of
-constraints are needed then it is recommended to use a gatekeeper pattern which
-involves electing a primary Crux node (e.g. using ZooKeeper) to execute
-transactions against, thereby creating a similar effect to Datomic's transactor
-component
+throughput of the entire system. Within Crux, transaction functions are installed
```

Comment:
Fairly unsure on this section - copied this across from the dev journal, not sure whether it entirely fits this section or not though.